Dockerfile

```diff
@@ -1,14 +1,32 @@
 # Base image
-FROM node:16.14.0 AS base
+FROM ubuntu:focal
 
-# Install git
-# Others are dependencies of our gyp dependencies
+# For tzdata
+ARG DEBIAN_FRONTEND=noninteractive
+ARG TZ=America/Los_Angeles
+
+# Autolink repository https://docs.github.com/en/packages/learn-github-packages/connecting-a-repository-to-a-package
+LABEL org.opencontainers.image.source=https://github.com/algolia/renderscript
+LABEL org.opencontainers.image.revision=$VERSION
+
+# === INSTALL Node.js ===
 RUN apt-get update && \
-  apt-get install -y --no-install-recommends \
-  g++ \
-  git \
-  make \
-  python
+  # Install node16
+  apt-get install -y curl wget && \
+  curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
+  apt-get install -y nodejs && \
+  # Feature-parity with node.js base images.
+  apt-get install -y --no-install-recommends git openssh-client && \
+  npm install -g yarn && \
+  # Install Python 3.8
+  apt-get install -y python3.8 python3-pip && \
```

Comment:
All this Python stuff is actually not needed. You can remove it. The main reason it is there as of today is because for Python users we use the same version. But in v1.21 Python gets its own image.

Reply:
Thanks, I'll try without 🙏🏻